src/main/scala/org/ergoplatform/ErgoLikeContext.scala

```diff
@@ -25,6 +25,9 @@ class ErgoLikeContext(val currentHeight: Height,
 
   def withTransaction(newSpendingTransaction: ErgoLikeTransactionTemplate[_ <: UnsignedInput]): ErgoLikeContext =
     ErgoLikeContext(currentHeight, lastBlockUtxoRoot, boxesToSpend, newSpendingTransaction, self, extension)
+
+  def withSelf(newSelf: ErgoBox): ErgoLikeContext =
```

Comment:
Why can we have need for this?

Reply:
@kushti I added it in some intermediate version, now it is not used. Remove?